Source/Scene/Cesium3DTileset.js

```diff
@@ -309,6 +320,31 @@ define([
         });
     };
 
+    function loadRefiningTiles(refiningTiles) {
+        // Tiles that use replacement refinement and have empty child tiles need to keep track of
+        // descendants with content in order to refine correctly.
+        var stack = [];
```

Comment:
Maybe not yet. While some code is duplicated, I think its easier to understand this way.